test/test_torch.py

```diff
@@ -14981,6 +14981,42 @@ def test_addmv(self, device, dtype):
 
         self.assertEqual(res1, res2, atol=self.precision)
 
+    @dtypesIfCUDA(*([torch.half, torch.float, torch.double]
+                    + ([torch.bfloat16] if TEST_WITH_ROCM else [])))
+    @dtypes(torch.float, torch.double)
+    def test_addmv_rowmajor_colmajor_incx_incy_lda(self, device, dtype):
+        # tests (o, s)*(s).  o is output size, s is summed size.
+        o = 5
+        s = 3
+        a_data = torch.arange(1, o * s, o * s - 1, device=device, dtype=dtype).view(o, s)
```

Comment:
arange is start, stop, step, step should be 1, so tests are legitimately failing.

Reply:
fixed.  my rebuilds were failing last night for unrelated reasons so i got impatient, flake8ed for syntax and pushed without running.  my fault.